### ISSUE TITLE
Introduce pagination on History component

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#2532]
I guess once https://github.com/kodadot/nft-gallery/pull/2523 is merged, 
would be nice to have pagination on the history component as lots of events are there. 

Mind adding URL parameters so results are shareable
- page
- a type of event is being shown

**Comment by Jarsen136** [comment]
I will work on this task.
